---
layout: page
---

<div align="center"><h3>Projects</h3></div>

---------------------------------------------
  
#### Neofunctionalization of tandem duplicate genes encoded novel putative b-L-arabinofuranosidases in *Arabidopsis thaliana*   
*	Examined raw RNA-seq data from six tissues of three plant species to reveal the neofunctionalization of two tandem duplicate genes, verified by further RNA-seq analysis using 2,843 samples from previous studies. 
*	Identified protein putative novel function encoded by tandem duplicate genes through analyses in domain, key residue, and protein structure comparisons.
*	Detected epigenetic modifications within tandem duplicate genesâ€™ regulatory regions via ChIP-seq and MN-Seq analysis.
*	Studied divergent phenotypic changes by tandem duplicate genes through plant growth traits extraction from automated plant rosette images.

[Analysis pipeline](https://github.com/tobytaogla/Athaliana_genus_duplicate_gene)

#### Optimization and Application of non-native *Phragmites australis* Transcriptome Assemblies   
*	*De novo* assembled 49 *P. australis* transcriptomes through four assembly tools and multiple parameters and selected the optimal one via comprehensive quality assessments.
*	Incorporated 13 strategies and six databases for annotating the selected transcriptome.
*	Steered downstream analyses, such as identifying differentially expressed transcripts and putative transcription factors as well as detecting simple sequence repeat markers to deliver a valued resource for future *P. australis* studies.  
[Analysis pipeline](https://github.com/tobytaogla/Phragmites-australis-transcriptome-optimal-assembly)   
[Results](https://fanlab.wayne.edu/resource/Phragmite_australis_transcriptome.html)

#### Languages and genes  
*	Identified the link between typological differences among languages with genetic differences in cooperation with communication sciences researchers.
*	Retrieved alleles frequency information via public databases and discovered SNPs linked to certain language features.
 <iframe width="560" height="315"
src="https://www.youtube.com/watch?v=Uu2feVLBsrM" 
frameborder="0" 
allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" 
allowfullscreen></iframe>

#### Evolutionary fates and dynamic functionalization of young duplicate genes in *Arabidopsis* genomes
*	Employed a phylogenetic approach with five tissues RNA-seq data from three closely related plant species for classification of genomic evolution of young duplicate genes. 
*	Highlighted relation in DNA methylation level, histone modification, and young duplicate genes evolutionary fates. 
   